EventFilter/L1TRawToDigi/plugins/implementations_stage2/ZDCUnpacker.cc

```diff
@@ -0,0 +1,65 @@
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "EventFilter/L1TRawToDigi/plugins/UnpackerFactory.h"
+
+#include "L1Trigger/L1TCalorimeter/interface/CaloTools.h"
+
+#include "L1TObjectCollections.h"
+
+// #include "L1TStage2Layer2Constants.h"
+#include "ZDCUnpacker.h"
+#include "GTSetup.h"
+
+namespace l1t {
+  namespace stage2 {
+    ZDCUnpacker::ZDCUnpacker() : EtSumZDCCopy_(0) {}
+
+    bool ZDCUnpacker::unpack(const Block& block, UnpackerCollections* coll) {
+      using namespace l1t::stage2;
+      uint32_t zdc_mask = 0x3FF;
+      LogDebug("L1T") << "Block ID  = " << block.header().getID() << " size = " << block.header().getSize();
+
+      int nBX = int(ceil(block.header().getSize() / zdc::nOutputFramePerBX));

```

Comment:
Please, remove `int(ceil(`, since it doesn't do anything on the result of an integer division.

If you need to round the result up, please change it to
```suggestion
      int nBX = (block.header().getSize() + zdc::nOutputFramePerBX - 1) / zdc::nOutputFramePerBX;
```